js/basic_js.js

```diff
@@ -0,0 +1,58 @@
+
+function getSalary(salary) {
+    salary==Number
+    if (n==15) console.log('thuế', 0.3*n*10**6, 'lương ròng', (15-0.3*n)*10**6 )  
+    else if (n==7) console.log('thuế', 0.2*n*10**6, 'lương ròng', (7-0.2*n)*10**6) 
+    else if(n>7 && n<15) console.log('thuế', 0.1*n*10**6, 'lương ròng', (n-0.1*n)*10**6)  
+}
+function getAge(age) {
+    age==Number
+    if (n < 16)
+        console.log('hoc sinh khong du dieu kien')
+    else console.log('hoc sinh du dieu kien')
+}
+function checkNumber(n) {
```

Comment:
2 block hoàn toàn khác nhau thì viết cách nhau ra
Đổi `n` thành 1 biến có ý nghĩa